src/main/java/nl/workingtalent/wtacademy/user/ReadUserDto.java

```diff
@@ -0,0 +1,101 @@
+package nl.workingtalent.wtacademy.user;
+
+import java.util.List;
+
+import java.util.stream.Collectors;
+
+public class ReadUserDto {
+
+	private long id;
+
+	private String firstName;
+
+	private String lastName;
+
+	private String email;
+
+	private Role role;
+
+	private List<Long> reviews;
+
+	private List<Long> reservations;
+
+	private List<Long> loans;
```

Comment:
Moeten dit geen strings zijn in read Dto? zodat je niet dat oneindige JSON krijgt?

Reply:
Je krijgt de id's terug van reviews, reservations en loans. Die zijn in de vorm van een Long. Omdat je geen objecten terug krijgt krijg je ook geen recursion. Is er een andere reden om er Strings van te maken?